public/style.css

```diff
@@ -0,0 +1,68 @@
+body {
```

Comment:
good work with the css and putting it into a separate style sheet. It took me far too long to work out how to do this!